bpf/bpf_host.c

```diff
@@ -1345,15 +1380,15 @@ int cil_to_netdev(struct __ctx_buff *ctx __maybe_unused)
 	}
 #endif
 
+	if (!validate_ethertype(ctx, &proto)) {
```

Comment:
This would have occurred 3 separate times in different header guards, so I hoisted it up and made it evaluate unconditionally, I think the code is cleaner this way.

Reply:
@joestringer this particular block of code has not changed from the previous commit, just moved. 

If we want to do refactoring of this block, I'd be happy too, but maybe we do this in a dedicated PR which addresses this concern directly?

Reply:
Going to resolve this as its purely cosmetic and this change is carried over from the previous code. Happy to address this in some other housekeeping commits outside of this PR tho.

Reply:
Yep makes sense, agree it's already the case with this code and doesn't need to be implemented here in this PR.